components/channel_header/__snapshots__/channel_header.test.jsx.snap

```diff
@@ -1366,13 +1426,23 @@ exports[`components/ChannelHeader should render properly when populated with cha
     />
     <HeaderIconWrapper
       ariaLabel={true}
-      buttonClass="style--none channel-header__icon"
+      buttonClass="style--none channel-header__icon wide"
       buttonId="channelHeaderPinButton"
       iconComponent={
-        <PinIcon
-          aria-hidden="true"
-          className="icon icon__pin"
-        />
+        <div
```

Comment:
isn't the pattern repeated too many times? maybe we could abstract it, originally it was small, but now that it has grown seems like a good option